src/OrbitGl/CaptureViewElement.cpp

```diff
@@ -170,6 +169,7 @@ CaptureViewElement::EventResult CaptureViewElement::HandleMouseEvent(
     }
   }
 
+  mouse_pos_cur_ = mouse_pos;
```

Comment:
It seems like `mouse_pos_cur_` can now only be inside the CaptureViewElement, doesn't that mean that the `mouse_pos_cur_` can be out of sync with the actual mouse position and a CaptureViewElement could think that the mouse is over it when it is not? Or is mouse_pos_cur_ cleared somehow? If that's handled properly, please disregard comment. Lgtm otherwise.

Reply:
Yes, we are handling it by its parent using MouseLeave events. :)